shesmu-server/src/main/java/ca/on/oicr/gsi/shesmu/input/nabu/NabuJsonRepository.java

```diff
@@ -38,14 +41,11 @@ protected NabuValue convert(ObjectNode node) {
 		return new NabuValue(//
 				node.get("fileswid").asInt(), //
 				node.get("filepath").asText(), //
-				node.get("qcstatus").asText(), //
+				status, //
 				node.get("username").asText(), //
-				node.get("comment").asText(), //
+				node.get("comment").asText(""), //
```

Comment:
Why the empty double quotes?

Reply:
That provides a default if missing or null and Shesmu can't do null, so empty string is close enough.